src/server/services/validate-upload.js

```diff
@@ -62,6 +62,32 @@ async function validateEcCode ({ upload, records, trns }) {
   }
 }
 
+async function validateVersion ({ upload, records, rules }) {
+  const logicSheet = records.find(doc => doc.type === 'logic').content
```

Comment:
```suggestion
  const logicSheet = records.find(record => record.type === 'logic').content
```

Nit: no reason to reintroduce the confusing term "doc" here